src/smt/proof_manager.cpp

```diff
@@ -105,6 +105,20 @@ PfManager::PfManager(Env& env)
 
 PfManager::~PfManager() {}
 
+// Remove in favor of `std::erase_if` with C++ 20+.
```

Comment:
I'd prefer not adding this. If you think we should update this when C++ 20 is available, I would open a cvc5 wishue and reference in this code here.

Reply:
Done!

Reply:
Are these wishues public? I can't work out where to find the discussion of the wishue.

It would be great if this `#if __cplusplus <= 202002L` could be restored to allow building with newer C++ toolchains.